testdata/recursive_thunk.golden

```diff
@@ -0,0 +1,50 @@
+RUNTIME ERROR: xxx
```

Comment:
These take up a lot of vertical space now.  Can it not be one line per trace (or at most 2 like Python)?

Reply:
Well, yes, but there's a trade-off. The more vertically concise it is, the harder it is to distinguish frames and information within a frame. Especially when we show which part of the line caused a problem (which I find extremely useful at times).  If we keep using only color to mark problematic part of a line 2 line python-line stack trace is definitely doable. However, for me personally vertical space is not that much of an issue here, for me it's more important that it's easy to visually scan and frames don't blend into each other. I wonder how people feel about it (and why). In compiler messages it seems the world is going towards more verbose and vertically-heavy, but static messages are generally consumed one at a time, so it's different.

Anyway, the current version is only a starting point, I haven't put much thought to it yet. As we polish it, we can quite easily try different variants and see what works best (or I can just make it python-like if you think that's the best option).

Also traditional one-line version is still available (pretty=false).